functions.php

```diff
@@ -197,14 +197,18 @@ function atomic_blocks_fonts_url() {
 		}
 	}
 
-	$query_args = array(
-		'family' => urlencode( implode( '|', $font_families ) ),
-		'subset' => urlencode( 'latin,latin-ext' ),
-	);
+	if ( !empty ( $font_families ) ) {
+
+		$query_args = array(
+			'family' => urlencode( implode( '|', $font_families ) ),
+			'subset' => urlencode( 'latin,latin-ext' ),
+		);
 
-	$fonts_url = add_query_arg( $query_args, "//fonts.googleapis.com/css" );
+		$fonts_url = add_query_arg( $query_args, "//fonts.googleapis.com/css" );
 
-	return $fonts_url;
+		return $fonts_url;
```

Comment:
The return line should always execute so that at the very least it returns the empty string it's set to at the top. Just move this below the bracket on 211.